src/styles/_mixins.scss

```diff
@@ -395,52 +395,7 @@
 
 /* NEW MIXINS */
 
-@mixin inputDefaults {
-  -webkit-appearance: none; // need this to allow box-shadow to apply on mobile
-  background-color: var(--kui-color-background, $kui-color-background);
-  border: 0;
-  border-radius: var(--kui-border-radius-30, $kui-border-radius-30);
-  box-shadow: var(--kui-shadow-border, $kui-shadow-border);
-  color: var(--kui-color-text, $kui-color-text);
-  font-family: var(--kui-font-family-text, $kui-font-family-text);
-  font-size: 16px; // needs to be at least 16px to prevent automatic zoom in on focus on mobile
-  font-weight: var(--kui-font-weight-regular, $kui-font-weight-regular);
-  line-height: var(--kui-line-height-40, $kui-line-height-40);
-  outline: none;
-  padding: var(--kui-space-40, $kui-space-40) $kInputPaddingX;
-  transition: box-shadow $kongponentsTransitionDurTimingFunc;
-  width: 100%;
-
-  @media (min-width: $kui-breakpoint-mobile) {
-    font-size: var(--kui-font-size-30, $kui-font-size-30);
-  }
-
-  &::placeholder {
-    color: var(--kui-color-text-neutral, $kui-color-text-neutral);
-  }
-}
-
-@mixin inputHover {
-  box-shadow: var(--kui-shadow-border-primary-weak, $kui-shadow-border-primary-weak);
-}
-
-@mixin inputFocus {
-  box-shadow: var(--kui-shadow-border-primary, $kui-shadow-border-primary), var(--kui-shadow-focus, $kui-shadow-focus);
-}
-
-// shared styles between disabled and readonly states
-@mixin inputDisabledReadOnly {
-  background-color: var(--kui-color-background-disabled, $kui-color-background-disabled);
-  box-shadow: var(--kui-shadow-border-disabled, $kui-shadow-border-disabled);
-  color: var(--kui-color-text-disabled, $kui-color-text-disabled);
-}
-
-@mixin inputDisabled {
-  @include inputDisabledReadOnly;
-
-  cursor: not-allowed;
-}
-
-@mixin inputReadOnly {
-  @include inputDisabledReadOnly;
-}
+/* stylelint-disable-next-line no-invalid-position-at-import-rule */
+@import "@/styles/mixins/inputs";
+/* stylelint-disable-next-line no-invalid-position-at-import-rule */
+@import "@/styles/mixins/labels";
```

Comment:
issue: these imports must go at the top (and remove the ignore comment)

Reply:
Fixed